src/app/components/app/index.js

```diff
@@ -18,7 +18,11 @@ export function App() {
 					<Sidebar />
 				</div>
 				<div className="bb-app__main">
-					<Route path="/block/:slug">
+					<Route path="/block/:slug/:currentTab">
+						<RouteBlock />
+					</Route>
+
+					<Route exact path="/block/:slug">
```

Comment:
This works but another way to deal with this could have been "nested routing". Removing the "exact" here and adding subroutes inside `RouteBlock`